src/mame/drivers/inorbubble.cpp

```diff
@@ -0,0 +1,62 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/****************************************************************************************
+    Skeleton driver for Inor "bubble" electromechanical machines.
+    Known games with this PCB:
+      -Olimpic Hockey (latest generation, there's an older version on different hardware)
+      -Futbol Champ
+
+    Hardware:
+      -8080
+      -MEA-8000 Voice Synth
+      -AY-3-8910
+      -MOSEL MS6516L-10PC 2Kx8 Static RAM
+****************************************************************************************/
+
+#include "emu.h"
+#include "cpu/i8085/i8085.h"
+#include "sound/ay8910.h"
+#include "sound/mea8000.h"
+#include "speaker.h"
+
+namespace {
+
+class inorbubble_state : public driver_device
+{
+
+public:
+	inorbubble_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+	{
+	}
+
+	void inorbubble(machine_config &config);
+
+private:
+	required_device<cpu_device> m_maincpu;
+};
+
+INPUT_PORTS_START(inorbubble)
+INPUT_PORTS_END
+
+void inorbubble_state::inorbubble(machine_config &config)
+{
+	// Basic machine hardware
+	I8080(config, m_maincpu, 4'000'000); // Unknown clock // AMD 8080 CPU, exact model unknown

```

Comment:
You said “the code seems to be Z80” in the description – if it’s using Z80 features, it won’t work on an 8080.

Reply:
That's why I'm asking for help, this is far beyond my skills.